server/sonar-server/src/test/java/org/sonar/server/organization/ws/CreateActionTest.java

```diff
@@ -109,14 +116,41 @@ public void verify_response_example() throws URISyntaxException, IOException {
   }
 
   @Test
-  public void request_fails_if_user_does_not_have_SYSTEM_ADMIN_permission() {
+  public void request_fails_if_user_does_not_have_SYSTEM_ADMIN_permission_and_logged_in_user_can_not_create_organizations() {
     expectedException.expect(ForbiddenException.class);
     expectedException.expectMessage("Insufficient privileges");
 
     executeRequest("name");
   }
 
   @Test
+  public void request_succeeds_if_user_has_SYSTEM_ADMIN_permission_and_logged_in_user_can_not_create_organizations() {
+    giveUserSystemAdminPermission();
+    mockForSuccessfulInsert(SOME_UUID, SOME_DATE);
```

Comment:
As `mockForSuccessfulInsert()` always mocks `System2.now()` with `SOME_DATE`, I suggest to use:
```
private System2 system2 = new TestSystem2().setNow(SOME_DATE);